ports/mimxrt/boards/MIMXRT1011.ld

```diff
@@ -2,6 +2,7 @@
 __stack_size__ = 0x6000;
 _estack = __StackTop;
 _sstack = __StackLimit;
+__heap_size__ = 0;
```

Comment:
Would be good to add a comment here explaining that the traditional C heap is unused.

Reply:
> Would be good to add a comment here explaining that the traditional C heap is unused.

Done.